crates/sovereign-sdk/full-node/db/sov-db/src/ledger_db/rpc.rs

```diff
@@ -428,6 +428,14 @@ impl LedgerRpcProvider for LedgerDB {
         }
     }
 
+    fn get_last_verified_proof(&self) -> Result<Option<VerifiedProofResponse>, anyhow::Error> {
```

Comment:
what if there wasn't a proof on the last L1 block?

this should return the *last* proof. maybe along with the L1 height it was found on

Reply:
maybe changing L1 heigh => verified proof table encoding to be seek encoding is enough to pull the last proof in that table.

Reply:
Done, now it uses seek to last to get the latest proof also returns l1 height with it.